cherrypy/test/test_static.py

```diff
@@ -215,13 +215,13 @@ def test_config_errors(self):
         self.assertErrorPage(500)
         if sys.version_info >= (3, 3):
             errmsg = (
-                'TypeError: staticdir\(\) missing 2 '
+                r'TypeError: staticdir\(\) missing 2 '
```

Comment:
Alright, let's leave slashes untouched, but I'm still curious about their original purpose. Maybe in some ancient Python 2 they were here for escaping.